examples/federated-learning-mistnet-client.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tensorflow/tensorflow:1.15.4
+
+RUN apt update \
+  && apt install -y libgl1-mesa-glx
+
+COPY ./lib/requirements.txt /home
+
+RUN pip install -r /home/requirements.txt
+
+ENV PYTHONPATH "/home/lib:/home/plato"
+
+COPY ./lib /home/lib
+COPY ./plato /home/plato
```

Comment:
better to add plato as the requirement of sedna-lib instead of expose to the user.

Reply:
Due to the large package issue, we currently add Plato as third-party lib.